scripts/installscripts/tests/test_installation.sh

```diff
@@ -220,16 +222,20 @@ verify_autohotspot_settings() {
     fi
 }
 
-verify_apt_packages(){
-    local phpver="$(ls -1 /etc/php)"
-    local packages="samba
-samba-common-bin gcc lighttpd php${phpver}-common php${phpver}-cgi php${phpver} at mpd mpc mpg123 git ffmpeg
-resolvconf spi-tools python3 python3-dev python3-pip python3-setuptools python3-wheel python3-mutagen python3-gpiozero
-python3-spidev netcat alsa-utils"
-    local packages_raspberrypi="raspberrypi-kernel-headers"
-    local packages_spotify="libspotify-dev mopidy mopidy-mpd mopidy-local mopidy-spotify libspotify12
-python3-cffi python3-ply python3-pycparser python3-spotify"
-    local packages_autohotspot="dnsmasq hostapd iw"
+call_with_apt_packages_from_file() {
+    local package_file="$1"
+    shift
+
+    # read line from the file and remove comments. Pass it over xargs as arguments to the default echo command.
+    sed 's/#.*//g' ${package_file} | xargs "$@"
```

Comment:
Isn't it better understandable, if you call here the apt-get command instead of using the passed arguments?!

I mean:
`sed 's/#.*//g' ${package_file} | xargs ${apt_get} install`

The name of the function already implies, that something with apt will happen

Reply:
I named it "apt" mainly to distinguish from the function for the "pip" packages.
Also the command given here is not `apt-get install` but `echo`, as the test only reads the file content.

Reply:
Oh right. That was just a copy paste fail.

Reply:
renamed the function to `call_with_args_from_file` and added a function usage comment